#! /bin/sh -e

# DP: Do not build the java runtime (from this source package)

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p0 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p0 < $0
        ;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1
esac
exit 0

--- configure.in~	2005-09-13 09:01:28.000000000 +0200
+++ configure.in	2005-11-21 14:08:20.691478720 +0100
@@ -171,7 +171,7 @@
 		target-libstdc++-v3 \
 		target-libmudflap \
 		target-libgfortran \
-		${libgcj} \
+		target-libffi \
 		target-libobjc \
 		target-libada"
 
--- configure~	2005-09-13 09:01:28.000000000 +0200
+++ configure	2005-11-21 14:08:44.191906112 +0100
@@ -937,7 +937,7 @@
 		target-libstdc++-v3 \
 		target-libmudflap \
 		target-libgfortran \
-		${libgcj} \
+		target-libffi \
 		target-libobjc \
 		target-libada"
 
